### ISSUE TITLE
Suggestion: More menu items

### ISSUE THREAD

**Issue by Emma-Walker** [BEEmod/BEE2.4#1294]
I don't know if it's even possible. But I ask anyway. Is it possible to have more items in the palette?

Like so:
![beemod-with-more-items](https://user-images.githubusercontent.com/8507237/80437844-662b9800-890b-11ea-8904-5672d894734b.jpg)

**Comment by Critfish** [comment]
Not possible. The puzzlemaker is hardcoded to only have 32 items. We can't change it.

**Comment by Emma-Walker** [comment]
Is it possible then to combine some more items, like Turret + Turret Dropper. Or High Energy Pellet Emitter + Catcher. Door + Chamber Lock... etc

**Comment by Critfish** [comment]
For some items, it's feasible to combine them. For others, it's simply not practical. Sure, you could *theoretically* collapse 2400 different items into one, but it'd be hardly practical. Some items behave completely differently than others and would cause issues if combined.

And no, we can't make new submenus. We can use existing ones(eg. Cube Type, Button Type), but we can't make new ones.

**Comment by Emma-Walker** [comment]
So many of the fizzlers could be combined and then the type of fizzler could be set just like the type of cube?

**Comment by Luke18033** [comment]
If we could expand the palette, don't you think we would have already done so? As for combining items, that ability is very limited. It is possible to combine fizzlers in pairs using the existing Hazard Type property (as I suggested in BEEmod/BEE2-items#1596), but other than that we've pretty much already combined everything that we can.

One important thing to note is that if you remove an item from the palette, existing copies of it will remain in maps. That allows you to swap between multiple different palettes containing all the items you want to use.

**Comment by Luke18033** [comment]
Also, how are you using 4.35 on Windows XP...?

**Comment by Emma-Walker** [comment]
> It is possible to combine fizzlers in pairs using the existing Hazard Type property (as I suggested in [BEEmod/BEE2-items#1596](https://github.com/BEEmod/BEE2-items/issues/1596))

That's a very good suggestion. One question though, isn't the preview (PeTI) always limited to the standard emancipation grid and laser? So even with combined Fizzlers all would look blue and red?

>One important thing to note is that if you remove an item from the palette, existing copies of it will remain in maps.  That allows you to swap between multiple different palettes containing all the items you want to use.

Good to know.

>Also, how are you using 4.35 on Windows XP...?

It's Windows 7 with Luna Theme :)

**Comment by Luke18033** [comment]
The field would always appear blue/red, yes, but that's just as much of a problem with the current custom fizzlers (in fact it's worse, since they're all blue). The preview on the emitter models was added for a reason.

As for the Windows theme, I wasn't aware it was possible to port that over.

**Comment by supersean08-pro** [comment]
> Also, how are you using 4.35 on Windows XP...?

in windows 7 theirs an option to change the theme, but in windows 10, they dumb it down
